### PR TITLE
[WFLY-17412] Create a testbom

### PR DESCRIPTION
**Pull request by parsharma**
Issue: https://issues.redhat.com/browse/WFLY-17412

### CODE REVIEW

**Comment by bstansberry** [comment]
Hi @parsharma -- please include 'WFLY-17412' in the commit message and in the title of the PR (which can be edited.) Thanks!

We'll need to discuss with @kabir how to deal with the 'Cloud Tests Trigger/run-test' failure. I assume it's a case of those tests depending on our root pom for their own test dependencies, so there will need to be a migration.

File: testbom/pom.xml [hunk 1]
```diff
@@ -0,0 +1,315 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bstansberry) [inline testbom/pom.xml:1]: Please move this directory into the boms directory.

Comment (bstansberry) [reply testbom/pom.xml:1]: Please call it boms/standard-test. Thanks!

Comment (parsharma) [reply testbom/pom.xml]: Done, thanks.

Comment (parsharma) [reply testbom/pom.xml]: > Hi @parsharma -- please include 'WFLY-17412' in the commit message and in the title of the PR (which can be edited.) Thanks!
> 
> We'll need to discuss with @kabir how to deal with the 'Cloud Tests Trigger/run-test' failure. I assume it's a case of those tests depending on our root pom for their own test dependencies, so there will need to be a migration.

yes, I have found out that some deps depend on root pom.xml.

**Comment by bstansberry** [comment]
Thanks @parsharma

**Pull request merged**